localenv/mock-account-servicing-entity/app/lib/accounts.server.ts

```diff
@@ -15,10 +15,10 @@ export interface Account {
 
 export interface AccountsServer {
   clearAccounts(): Promise<void>
-  setPaymentPointer(
+  setWalletAddress(
     id: string,
     pointerID: string,
-    paymentPointer: string
+    walletAddress: string
   ): Promise<void>
```

Comment:
looks like `pointerID` is short for `paymentPointerID` and should be renamed as well

Reply:
@njlie I think this one still needs renaming